mitodl/micromasters
# remove dnt-policy.txt

#### What are the relevant tickets?

fixes #2636

#### What's this PR do?

- removes the dnt-policy.txt file that was included in #2880
- configures nginx (in dev & heroku) to return a 204 instead of a 404

#### How should this be manually tested?

Hit `./well-known/dnt-policy.txt` on a local version, and on a Heroku PR build, to see if it returns a 204

#### What best describes this PR or how it makes you feel?

Don't approve PRs over Slack.

![img](http://media.giphy.com/media/5zkHiL3b6OuAw/giphy.gif)
